plugins/source/postgresql/client/spec.go

```diff
@@ -0,0 +1,14 @@
+package client
+
+type Spec struct {
+	ConnectionString string   `json:"connection_string,omitempty"`
```

Comment:
What happens if you specify the same database as source and destination? Should we disallow that?

Reply:
I think we can't know in the source to which destination it is going, so it's up to the user really.

Reply:
Yeah, probably not something we can really prevent